### DISCUSSION TITLE
Upstream Improvements

### DISCUSSION THREAD

**Discussion by belfhi** [eduMFA/eduMFA#91] [General]
Hi,
just wanted to know why this is a _fork_ of the PrivacyIDEA repo? Couldn't the dependabot integration and the Passkey functionality have been implemented in the upstream repo? I like those improvements a lot but we also depend on other privacyidea components (e.g. keycloak-provider) so switching isn't so straightforward.
I see a lot of activity here, which is nice. Since these are _very_ small projects I think it makes most sense to bundle all the efforts into one open source project. 
Has there been any communication with NetKnights? In my experience, the developers are open to contributions (even if my PR was just a docs fix). 
-johannes

**Comment by johannwagner** [comment]
Hey, 

please see some of the press informations regarding this topic.

https://www.fu-berlin.de/presse/informationen/fup/2024/fup_24_057-edumfa-hochschulen/index.html

Best, 
  Johann

**Reply by johannwagner** [reply]
This does not answer all of your questions, sadly. There is a whole other background story between the associated universities and NetKnights which is not discussed publicly. Therefore, to fix things and move forward, the fork was necessary. 

Best, Johann

**Reply by belfhi** [reply]
I was going to say, the press release is a little "biased". So I thought clearly there was a backstory^^
Well that's frustrating. We depend on the Keycloak integration so we don't really want to switch.
